app/controllers/concerns/joiner.rb

```diff
@@ -60,10 +60,10 @@ def join_room(opts)
       opts[:mute_on_start] = room_setting_with_config("muteOnStart")
 
       if current_user
+        opts[:avatarURL] = current_user.image
```

Comment:
@farhatahmad , do we need to check if current_user.image is present here? It seems unnecessary to me.